airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresQueryUtils.java

```diff
@@ -73,7 +73,7 @@ public record ResultWithFailed<T>(T result, List<io.airbyte.protocol.models.v0.A
       """;
   public static final String MAX_CURSOR_VALUE_QUERY =
       """
-        SELECT %s FROM %s WHERE %s = (SELECT MAX(%s) FROM %s);
+        SELECT "%s" FROM %s WHERE "%s" = (SELECT MAX("%s") FROM %s);
```

Comment:
verified by looking at `NULL_CURSOR_VALUE_WITH_SCHEMA_QUERY` and `NULL_CURSOR_VALUE_NO_SCHEMA_QUERY` that column names are enclosed in double quotes. LGTM.